templates/i/comment/thread.tx

```diff
@@ -72,6 +72,9 @@
 		<img id="img-<: $screenshot.media.id:>" src="/static/img/spinner.gif" data-src="<: $screenshot.media.url :>" alt="<: $screenshot.media.upload_filename :> - Full" class="reveal-modal reveal-modal--img" />
 	<: } :>
 <: } :>
+: if ( $_.context == 'DDGC::DB::Result::Thread' || $_.context == 'DDGC::DB::Result::Idea' ) {
+    <h3>This forum has been archived</h3>
```

Comment:
If possible it would good to add a redirection/thanks message as well, e.g.:

```html
<p>Thank you all for the many comments, questions and suggestions. Particular thanks go to <a href="https://duck.co/user/x.15a2">user x.15a2</a> for constantly monitoring, replying and helping so many users here. To continue these discussions, please head over to the <a href="https://www.reddit.com/r/duckduckgo/">DuckDuckGo subreddit</a>.</p>
```